Servers/Administration/OJS.Servers.Administration/Controllers/ProblemsController.cs

```diff
@@ -430,16 +430,16 @@ await this.contestsValidationHelper
             IsReadOnly = true,
         });
 
-        if (!contest.IsOnlineExam)
+        formControls.Add(new FormControlViewModel
         {
-            formControls.Add(new FormControlViewModel
-            {
-                Name = AdditionalFormFields.ProblemGroupType.ToString(),
-                Options = EnumUtils.GetValuesFrom<ProblemGroupType>().Cast<object>(),
-                Type = typeof(ProblemGroupType),
-                Value = entity.ProblemGroup?.Type ?? default(ProblemGroupType),
-            });
+            Name = AdditionalFormFields.ProblemGroupType.ToString(),
+            Options = EnumUtils.GetValuesFrom<ProblemGroupType>().Cast<object>(),
+            Type = typeof(ProblemGroupType),
+            Value = entity.ProblemGroup != null ? entity.ProblemGroup.Type ?? default(ProblemGroupType) : default,
```

Comment:
What was wrong with the previous logic for `Value`?

Reply:
If problemGroup already has a default value (null). This is causing an error.

Reply:
How `entity.ProblemGroup?.Type ?? default(ProblemGroupType),` is causing an error? Doesn't the `?` in `entity.ProblemGroup?.` prevent it?

Reply:
With the new changes now works appropriately. Reverting the logic.